mne/forward/forward.py

```diff
@@ -875,6 +875,70 @@ def write_forward_meas_info(fid, info):
     end_block(fid, FIFF.FIFFB_MNE_PARENT_MEAS_FILE)
 
 
+def _check_loose(forward, loose):
+    """Check loose parameter input."""
+    if is_fixed_orient(forward):
```

Comment:
I added `and loose is not None`, so the warning will now only appear if loose is not None